controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/CuratorDb.java

```diff
@@ -67,7 +73,23 @@
 
     @Inject
     public CuratorDb(Curator curator) {
+        this(curator, curator.zooKeeperEnsembleConnectionSpec());
+    }
+
+    CuratorDb(Curator curator, String zooKeeperEnsembleConnectionSpec) {
         this.curator = curator;
+        this.zooKeeperEnsembleConnectionSpec = zooKeeperEnsembleConnectionSpec;
+    }
+
+    /** Returns all hosts configured to be part of this ZooKeeper cluster */
+    public List<HostName> cluster() {
+        return Arrays.stream(zooKeeperEnsembleConnectionSpec.split(","))
+                     .filter(hostPort -> !hostPort.isEmpty())
```

Comment:
Strange choice of variable name :)

Reply:
I suppose it means _host and port_ -- how about camel-casing that, or just using _host_? I read this as _port of the host_, and was confused.

Reply:
Fixed.